lib/storage_tables/attachable/one.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+module StorageTables
+  module Attachable
+    # Representation of a single attachment to a model.
+    class One < ActiveStorage::Attached::One
+      # Attaches an +attachable+ to the record.
+      #
+      # If the record is persisted and unchanged, the attachment is saved to
+      # the database immediately. Otherwise, it'll be saved to the DB when the
+      # record is next saved.
+      #
+      #   person.avatar.attach(params[:avatar]) # ActionDispatch::Http::UploadedFile object
+      #   person.avatar.attach(params[:signed_blob_id]) # Signed reference to blob from direct upload
+      #   person.avatar.attach(io: File.open("/path/to/face.jpg"), filename: "face.jpg", content_type: "image/jpeg")
+      #   person.avatar.attach(avatar_blob) # ActiveStorage::Blob object
+      def attach(attachable, filename:)
```

Comment:
De filename is nu een verplicht veld geworden terwijl deze eerder afgeleidt werd van `attachable`. In principe is het alleen vereist deze mee te geven als de attachable een blob is, in de andere gevallen zouden we nog steeds de filename moeten kunnen afleiden van de attachable.

Is niet een heel groot punt dit, kan eventueel naar een los issue voor later.